admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +179,174 @@ A failure from this check indicates that a significant amount of space in etcd
+|This set of checks verifies that Curator, Kibana, Elasticsearch, and Fluentd
+pods have been deployed and are in a `running` state, and that a connection can
+be established between the control host and the exposed Kibana URL. These checks
+will only run if the `openshift_hosted_logging_deploy` inventory variable is set
+to `true`, to ensure that they are executed in a deployment where
+xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[cluster logging] has been enabled.
+
+|`logging_index_time`
+|This check detects higher than normal time delays between log creation and log
+aggregation by Elasticsearch in a logging stack deployment. It fails if a new
+log entry cannot be queried through Elasticsearch within a timeout (by default,
+30 seconds). The check only runs if logging is enabled.
 
+A user-defined timeout may be set by passing the
+`openshift_check_logging_index_timeout_seconds` variable. For example, setting
+`openshift_check_logging_index_timeout_seconds=45` will cause the check to fail
+if a newly-created log entry is not able to be queried via Elasticsearch after
+45 seconds.
+
+|===
+
+[NOTE]
+====
 A similar set of checks meant to run as part of the installation process can be
 found in
 xref:../install_config/install/advanced_install.adoc#configuring-cluster-pre-install-checks[Configuring Cluster Pre-install Checks]. Another set of checks for checking certificate
 expiration can be found in
 xref:../install_config/redeploying_certificates.adoc#install-config-redeploying-certificates[Redeploying Certificates].
+====
+
+[[admin-guide-health-checks-via-ansible-playbook]]
+=== Running Health Checks via ansible-playbook
+
+To run the *openshift-ansible* health checks using the `ansible-playbook`
+command, specify your cluster's inventory file and run the *_health.yml_*
+playbook:
+
+----
+# ansible-playbook -i <inventory_file> \
+ifdef::openshift-enterprise[]
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-checks/health.yml
+endif::[]
+ifdef::openshift-origin[]
+    ~/openshift-ansible/playbooks/byo/openshift-checks/health.yml
+endif::[]
+----
+
+To set variables in the command line, include the `-e` flag with any desired
+variables in `key=value` format. For example:
+
+----
+# ansible-playbook -i <inventory_file> \
+ifdef::openshift-enterprise[]
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-checks/health.yml
+endif::[]
+ifdef::openshift-origin[]
+    ~/openshift-ansible/playbooks/byo/openshift-checks/health.yml
+endif::[]
+    -e openshift_check_logging_index_timeout_seconds=45
+    -e etcd_max_image_data_size_bytes=40000000000
+----
+
+To disable specific checks, include the variable `openshift_disable_check` with
+a comma-delimited list of check names in your inventory file before running the
+playbook. For example:
+
+----
+openshift_disable_check=etcd_traffic,etcd_volume
+----
+
+Alternatively, set any checks you want to disable as environment variables with
```

Comment:
Technically these aren't environment variables... I think you can safely `s/environment//`

On a docker command line, `-e` sets an environment variable. On an ansible command line, `-e` sets an extra variable or inventory variable. They look the same but they're accessed in completely different ways...